PROJECT

```diff
@@ -41,7 +41,43 @@ resources:
   domain: kiegroup.org
   group: app
   kind: KogitoInfra
-  path: github.com/kiegroup/kogito-operator/apis/v1beta1
+  path: github.com/kiegroup/kogito-operator/apis/app
```

Comment:
this naming can be improved imho, for product you have used rhpam,why not kogito for community?

Reply:
`app` and `rhpam` is api group. `app` group for community and `rhpam` group is for product.

Reply:
yes, I understand that, my question is why not rename it to avoid confusion? imho app is to vague.

Reply:
@ricardozanini @sutaakar @radtriste wdyt?

Reply:
Rename would affect group used in Kogito CRs, right? In such case it would be backward incompatible change.
I would rather prefer keeping the current group in such case, can be adjusted if we will switch to new CRD version.

Reply:
@sutaakar good point, I agree with you, @vaibhavjainwiz maybe file a jira to keep track of it when we are going to update the api version.

Reply:
created below jira to change group name
https://issues.redhat.com/browse/KOGITO-6064